frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentAccordionBody/StrategyDraggableItem/StrategyItem/CopyStrategyIconMenu/CopyStrategyIconMenu.tsx

```diff
@@ -150,7 +150,7 @@ export const CopyStrategyIconMenu: VFC<ICopyStrategyIconMenuProps> = ({
                     'aria-labelledby': `copy-strategy-icon-menu-${strategy.id}`,
                 }}
             >
-                {environments.map((environment) => {
+                {[...environments, environmentId].map((environment) => {
```

Comment:
So the environments that come through as the function parameter are "all other environments", but not this one, yeah? So we're just adding the ID of this one to the list. 

Edit: right, an environment's **name** is also it's ID, right?

Reply:
I tracked the code invocations and we pass current env and other envs as two different props. I merged those two with the current one forced to be the last one on the list. name/id is the same thing for environments